internal/sessiontest/session_test.go

```diff
@@ -514,3 +520,49 @@ func TestPOSTSizeLimit(t *testing.T) {
 	require.NoError(t, json.Unmarshal(bts, &rerr))
 	require.Equal(t, "http: request body too large", rerr.Message)
 }
+
+func TestDisableBinding(t *testing.T) {
```

Comment:
The body of this function looks a lot like `sessionHelperWithBinding`, which looks a lot like `sessionHelper`. Is it possible to decrease the code duplication?